### PR TITLE
feat(eibc): add more info to demand order updated event

### PR DESCRIPTION
**Pull request by zale144**
## Description

----

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Updated the scripts for local run, e.g genesis_config_commands.sh if the PR changes parameters
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

----

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: proto/dymensionxyz/dymension/eibc/events.proto [hunk 1]
```diff
@@ -12,24 +12,22 @@ option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
...
   // order_id is the unique identifier of the demand order.
   string order_id = 1;
-  // fulfillment_id is the unique identifier of the fulfillment.
+  // price is the price of the demand order.
   string price = 2;
-  // fulfillment_amount is the amount of the fulfillment.
+  // fee is the fee of the demand order.
   string fee = 3;
-  // is_fulfilled is the flag indicating whether the order is fulfilled.
-  bool is_fulfilled = 4;
```
Comment (omritoptix) [inline proto/dymensionxyz/dymension/eibc/events.proto:20]: why did we remove it? old clients don't need it?
also shouldn't we deprecate it? seems like we've changed it.

File: x/eibc/types/demand_order.go [hunk 2]
```diff
@@ -82,7 +82,6 @@ func (m *DemandOrder) GetCreatedEvent(proofHeight uint64) *EventDemandOrderCreat
 		OrderId:      m.Id,
 		Price:        m.Price.String(),
 		Fee:          m.Fee.String(),
-		IsFulfilled:  m.IsFulfilled(),
```
Comment (omritoptix) [inline x/eibc/types/demand_order.go:85]: why did we remove it? old client don't need it?

**Comment by omritoptix** [comment]
@zale144 gonna leave it open until you tested it with eibc-client and give your final approval

File: proto/dymensionxyz/dymension/eibc/events.proto [hunk 3]
```diff
@@ -48,6 +49,14 @@ message EventDemandOrderFeeUpdated {
...
   // new_fee is the new fee amount set in the order.
   string new_fee = 2;
+  // price is the price of the demand order.
+  string price = 3;
+  // packet_status is the status of the packet.
+  string packet_status = 4;
+  // rollapp_id is the id of the rollapp.
+  string rollapp_id = 5;
+  // proof_height is the height of the block when order was created.
+  uint64 proof_height = 6;
```
[the remark this answers is not in the document]
Comment (zale144) [reply proto/dymensionxyz/dymension/eibc/events.proto:59]: the eibc-client sees updated orders as new orders, so it needs extra information, as it gets from the created order.

**Review by omritoptix — APPROVED** [review]

**Pull request merged**